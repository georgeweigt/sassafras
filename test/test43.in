* log function ;

data ;
input log y ;
logy = log(y) ;
datalines ;
1 10
2 20
3 30
;
proc print ;
