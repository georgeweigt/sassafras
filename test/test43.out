                         Obs     LOG      Y       LOGY
                           1       1     10     2.3026
                           2       2     20     2.9957
                           3       3     30     3.4012

